content/release-notes/1.14.0.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2020-03-18T06:29:39-08:00
+linktitle: "1.14.0"
+title: "1.14.0"
+kubernetes: "1.15, 1.16, and 1.17"
+---
+
+{{<features>}}
+* Added support for a new `additionalNamespaces` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest.
+* Added a `requireMinimimalRBAC` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest. When enabled and running `kots install`, KOTS will default to a namespaced-scoped RBAC role instead of a cluster-scoped role.
+* Added a new `Distribution` template function to return a string indicating the managed Kubernetes offering (i.e. `openshift`, `eks`, etc).
+* Added a new [kURL Context](/reference/template-functions/kurl-context) and corresponding template functions to read 
+values from the kURL Installer CRD. These functions are not available on the config page.
+{{</features>}}
+
+{{<changes>}}
+* Changed the internal API that `kots upload` and `kots download` use to the Go API. The Typescript implementations are deprecated.
+* Added Velero status to the dashboard tile to identify if Velero is not installed and Snapshots are enabled.
```

Comment:
This should have a "When snapshots are enabled"

Reply:
`* Added Velero status to the dashboard tile when Snapshots are enabled to identify if Velero is not installed.`?